src/components/header.astro

```diff
@@ -8,21 +8,26 @@ const navItems: Array<NavItem> = [
   { title: "FAQs", url: "#faqs" },
   { title: "AstroCommunity", url: "https://git.astronvim.com/AstroCommunity" },
   { title: "Documentation", url: "https://docs.astronvim.com" },
+  {
+    title: "Github",
+    icon: "tabler:brand-github",
+    url: "https://git.astronvim.com",
+  },
   { title: "Socials", icon: "tabler:at", url: "#socials" },
 ];
 ---
 
 <header
   id="page-header"
-  class="absolute bottom-0 z-20 flex w-full items-center justify-between border-b border-transparent px-8 py-4 text-white"
+  class="sticky top-0 z-20 flex w-full items-center justify-between border-b border-default bg-default px-8 py-4 text-default"
 >
   <a
     class="flex items-center gap-3 hover:!text-default"
     href="#"
     aria-label="AstroNvim"
   >
-    <img src="/logo/astronvim.svg" alt="AstroNvim logo" class="h-10" />
-    <h1 class="hidden h-6 font-bold md:block">AstroNvim</h1>
+    <img src="/logo/astronvim.svg" alt="AstroNvim logo" class="h-5" />
+    <h1 class="hidden h-4 font-bold leading-none md:block">AstroNvim</h1>
```

Comment:
Why is `h1` hidden?
Wondering wouldn't it be better to have title next to a logo to improve "brand recognition" by strengthening association? Also could make it a bit more balanced visually

Reply:
@Infonautica The h1 is currently hidden on mobile and shown as soon as you hit the `md` breakpoint. I did not change this behaviour, I only touched its size and alignment.

Reply:
Yeah, this is intended. It's too large for smaller screens so we hide it.